src/Orleans.Runtime/Streams/QueueBalancer/LeaseBasedQueueBalancer.cs

```diff
@@ -256,11 +258,12 @@ private void CalculateResponsibility()
             {
                 activeBuckets = GetActiveSiloCount(this.siloStatusOracle);
             }
-            this.minimumResponsibilty = this.allQueues.Count / activeBuckets;
+            activeBuckets = Math.Max(1, activeBuckets);
+            this.minimumResponsibility = Math.Max(1, this.allQueues.Count / activeBuckets);
```

Comment:
I think we should remove this Math.Max here. If this.allQueues.Count/activeBUckets <1, then minResponsibility should be 0, but we still have maxResponsibility to be 1. so each silo will try to get 1 if they can. 
if min is 1, then max is 2, then each silo will do their best to get that 1, while they can really just settle for 0. It is just a matter of aggressiveness. does this make sense?

Reply:
yup